src/ccl/charts/chart.template.html

```diff
@@ -0,0 +1,14 @@
+<div class="chart" *ngFor="let chart of chartList">
```

Comment:
Yeah, mabe lets grab a conf room after scrum and draw out the view hierarchy on a whiteboard. Might help to see it.